example/package-lock.json

```diff
@@ -1356,41 +1351,13 @@
           "requires": {
             "@babel/runtime": "^7.12.5",
             "@testing-library/dom": "^8.0.0",
-            "@testing-library/jest-dom": "^5.11.6",
```

Comment:
can we get this lil fella outta here? like the whole file? not sure why `package-lock.json` decided to come along for the ride